packages/tools/replay-tool/src/replayMessages.ts

```diff
@@ -514,20 +516,16 @@ export class ReplayTool {
 
     private async mainCycle() {
         const originalSummaries =
-            this.args.testSummaries ? [...this.mainDocument.originalSummarySequenceNumbers] : [];
-        let nextSnapPoint;
-        do {
-            nextSnapPoint = originalSummaries.shift() ?? this.args.from;
-        } while (nextSnapPoint < this.args.from);
+            this.args.testSummaries
```

Comment:
Is this a change in behavior or just code restructuring?

Reply:
just code restructuring. i actually wrote this code long ago and it was so bad i had to make it better :)

Reply:
Haha okay. I get the same feeling with some of my code :-).